CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# 2.6.0 - 2024.06.19
+
+ - Support renaming of BitBake variables in Python and Shell functions
+ - Support completion on BitBake variable expansions in Shell functions
```

Comment:
If we want to be more precise, we added completion, hover, references and definitions (and highlighting) to what tree-sitter-bash calls "simple variable expansion" (`$foo`). I failed to find [the official terminology](https://www.gnu.org/software/bash/manual/html_node/Shell-Expansions.html) for those.

Reply:
Updated to "Detect BitBake variables in Shell functions"

Reply:
to me it does not seems obvious what these variable are, but I guess it is good enough